Source/CarthageKit/Xcode.swift

```diff
@@ -172,6 +172,11 @@ public struct BuildArguments {
 		// Disable code signing requirement for all builds
 		// Frameworks get signed in the copy-frameworks action
 		args += [ "CODE_SIGNING_REQUIRED=NO", "CODE_SIGN_IDENTITY=" ]
+		
+		// Allow to force unsanitized environment
```

Comment:
Starting Xcode 7, the UseSanitizedBuildEnvironment defaults to YES, which overrides/disables any externally set environment variables.

This env var allow a user to override the default behavior (=YES) in order to force xcodebuild to use the current environment. 

Is that clear enough? I've added that a while ago, and don't really recall what the status was on the issue/resolution we wanted to apply to the original issue.

Any guidance appreciated